modules/administration-guide/partials/proc_restoring-a-che-instance-from-a-backup-using-the-checlusterrestore-object.adoc

```diff
@@ -0,0 +1,66 @@
+[id="restoring-a-{prod-id-short}-instance-from-a-backup-using-the-checlusterrestore-object_{context}"]
+= Restoring a {prod-short} instance from a backup using the CheClusterRestore custom object
+
+You can restore a {prod-short} instance from a backup snapshot using custom resources. A new `CheClusterRestore` custom object is required every time you restore a {prod-short} instance from a backup snapshot; that is, editing an already consumed `CheClusterRestore` custom object, during or after backing up, has no effect.
+
+include::snip_warning-backup-snapshots-are-bound-to-specific-cluster.adoc[]
+
+NOTE: The `CheClusterRestore` custom object is unusable for recovering a {prod-short} instance of a different version of {prod-short}. To recover a {prod-short} instance of an earlier version of {prod-short} by using `{prod-cli}`, see xref:restoring-a-backed-up-instance-of-an-earlier-version-of-{prod-id-short}_{context}[].
+
+.Prerequisites
+
+* Backup snapshot of a {prod-short} instance on any of the following:
+** xref:backups-of-che-instances-to-an-sftp-backup-server.adoc[External SFTP backup server]
+** xref:backups-of-che-instances-to-amazon-s3.adoc[Amazon S3]
+** xref:backups-of-che-instances-to-a-rest-backup-server.adoc[External REST backup server]
+** xref:backups-of-che-instances-to-the-internal-backup-server.adoc[Internal backup server]
```

Comment:
It doesn't matter if the server is external or internal. Just a supported backup server with the snapshot to restore to is required.